libraries/lps/include/mcrl2/lps/lps_rewriter_type.h

```diff
@@ -42,6 +43,10 @@ lps_rewriter_type parse_lps_rewriter_type(const std::string& type)
   {
     return condition_one_point;
   }
+  if (type == "dataspec-prune")
```

Comment:
Jan Friso and I were still discussing the name of this option, but perhaps `prune-dataspec` might be preferable?